modules/apps/portal-odata/portal-odata-impl/src/test/java/com/liferay/portal/odata/internal/filter/FilterParserImplTest.java

```diff
@@ -77,6 +77,51 @@ public void testParseNonexistingField() {
 			filterString);
 	}
 
+	@Test
+	public void testParseSuboperationWithDateAndDuration()
```

Comment:
The name doesn't follow the same pattern that the other tests in this class.

Reply:
changed